scene/resources/font.cpp

```diff
@@ -495,6 +496,38 @@ Size2 Font::get_string_size(const String &p_string) const {
 
 	return Size2(w, get_height());
 }
+
+Size2 Font::get_wordwrap_string_size(const String &p_string, float p_with) const {
+
+	ERR_FAIL_COND_V(p_with <= 0, Vector2(0, get_height()));
+
+	int l = p_string.length();
+	if (l == 0)
+		return Size2(p_with, get_height());
+
+	float line_w = 0;
+	float h = 0;
+	float space_w = get_char_size(' ').width;
+	Vector<String> lines = p_string.split("\n");
```

Comment:
If we're splitting on `\n`. That's not technically word wrap, that's forced new lines. Perhaps this function could be renamed?

Reply:
if you dont split on \n then you would have the new line on the same line, since this function is meant to know how tall your rect needs to be to fit text it makes sense to split them.